WordPress/src/main/java/org/wordpress/android/AppInitializer.kt

```diff
@@ -663,8 +661,8 @@ class AppInitializer @Inject constructor(
             // Make sure the Push Notification token is sent to our servers after a successful login
             gcmRegistrationScheduler.scheduleRegistration()
 
-            // Force a refresh if user has logged in. This can be removed once we start using an anonymous ID.
-            exPlat.forceRefresh()
+            // Possible way to initialize variations repository. Not tested, verify when working on experiments.
+            // variationsRepository.initialize("${tracker.anonID}")
```

Comment:
I don't implement it because the project doesn't run any experiments, and what we consider "experiment id" can vary from use case. This is just a hint where we should likely initialize `VariationsRepository`.